classes/plugins/ThemePlugin.inc.php

```diff
@@ -571,6 +571,16 @@ public function getOptionValues($contextId) {
 						break;
 				}
 			}
+			// If the value isn't null and it's a multilingual field, then we must ensure it's an array
+			if ($optionConfig->isMultilingual && $value !== null && !is_array($value)) {
```

Comment:
This is what's missing to make the multilingual field works. Its content is stored as JSON, but only fields that have a `default` value (see `switch()` above) are being decoded.